hugo-extended.yaml

```diff
@@ -0,0 +1,31 @@
+package:
+  name: hugo-extended
+  version: 0.112.3
+  epoch: 0
+  description: The world's fastest framework for building websites.
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    # required for extended SASS libraries
+    # - libc6-compat
```

Comment:
`libc6-compat` isn't available yet, may be required per: https://github.com/gohugoio/hugo/blob/3297b395d8d5fadb45b9055e55458967fb9f4a76/Dockerfile#L33